module/input_handler.py

```diff
@@ -57,7 +68,9 @@ def handle_new_input(self, input_commands) -> None:
                 help_parameters = input_commands[i + 1:]
                 self.handle_help(help_parameters)
                 break
-
+            if user_word.upper() == "SET" or user_word.upper() == "SELECT":
```

Comment:
this should be in global_commands instead

Reply:
It can, but the select function requires additional parameters. So just like help it is most likely easier to use a new function.

Reply:
For now this should be fine, if there are more exceptions like help and this we need to rethink this part.